wps-common/src/main/java/au/org/aodn/aws/util/AWSBatchUtil.java

```diff
@@ -137,9 +150,29 @@ public static JobDetail getJobDetail(AWSBatch batchClient, String jobId) {
 
             jobDetails = AWSBatchUtil.getJobDetails(batchClient, jobIds);
 
-            return jobDetails;
+            return sortByTimestamp(jobDetails);
         }
 
         return null;
     }
+
+
+    private static List<JobDetail> sortByTimestamp(List<JobDetail> jobList) {
+        JobDetail temp;
+        for(int step=0; step < jobList.size() - 1; ++step) {
```

Comment:
Should we leverage the java Collections.sort sorting functionality instead?  Its been well tested.

Reply:
We could - as an improvement?  ;)